app/models/types/patterns/collection.rb

```diff
@@ -32,8 +32,13 @@ module Types
   module Patterns
     Collection = Data.define(:patterns) do
       extend Dry::Monads[:result]
+      include Dry::Monads[:result]
```

Comment:
We needed the extend due to the rescue block returning a `Failure` as anything that relies on `.build` expects.

If you want to return the null value, it needs to come wrapped in a `Success`.

Reply:
I am pretty new to the concept of monads. Does it mean that as soon as I claim to be a monad all my methods need to respond in `Success` and `Failure` wrappers?

I thought it's fine to only do that for certain methods, because e.g. `self.null` has no way of failing, so it's a matter of always calling it like `Patterns.null.value!`, which looks like busy work to me.

(also there will be lots of methods defined that still don't abide to this... let's talk about `to_s` or `nil?` :D)

Reply:
Resolving this thread. We talked about this IRL.

Outcome:
* The `include` is a left-over from using `Success`/`Failure` from an instance method and could be removed by now
* It's totally fine that `null`/`empty` returns a value directly, without a wrapper, when it's always succeeding